portfolio/src/main/java/com/google/sps/data/MuseumLocations.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** Represents museum locations at specific lat and lng points. */
+public class MuseumLocations {
+    private String name;
+    private double lat;
+    private double lng;
+    private int layer;
+
+    public MuseumLocations(String name, double lat, double lng, int layer) {
+        this.name = name;
+        this.lat = lat;
+        this.lng = lng;
+        this.layer = layer;
```

Comment:
what does layering do?

Reply:
it positions the map location pins in such a way such that if there are 2 locations that are near each other, one pin is "layered" on top of another in the order that is defined